uc_order/uc_order.entity.inc

```diff
@@ -152,27 +145,28 @@ class UcOrderProduct extends Entity {
     return 'uc_order_product';
   }
 
-  /**
-   * Implements EntityInterface::bundle().
-   */
-  public function bundle() {
-    // return $this->type;
-  }
-
   /**
    * Implements EntityInterface::label().
    */
   public function label() {
-    return $this->title;
+    // Since uc_order_product entities are derived from nodes, we return the
+    // title of its "parent" node.
+    $node = node_load($this->order_product_id);
+    if ($node) {
+    return $node->title;
+    }
+    else {
+      return '';
```

Comment:
Since uc_order_product entities will always have a title property, this is unnecessary.